redis.go

```diff
@@ -135,7 +142,24 @@ func (c *baseClient) WrapProcess(fn func(oldProcess func(cmd Cmder) error) func(
 }
 
 func (c *baseClient) Process(cmd Cmder) error {
-	return c.process(cmd)
+	start := time.Now()
+	span, ctx := opentracing.StartSpanFromContext(c.context, "go_redis_client")
+	defer span.Finish()
+	span = span.SetTag("db.type", "redis")
+	span = span.SetTag("db.statment", cmd.String())
+	span = span.SetTag("span.kind", "client")
+
+	err := c.process(cmd)
+	if err != nil {
+		span.LogEvent("error")
+		span.LogKV(
+			"event", "error",
+			"type", err.Error(),
+			"latency", time.Since(start),
```

Comment:
latency ini simpan dalam seconds aja

Reply:
waktu itu bukannya semua latency di jadiin ms yah mas?

Reply:
latency sekon, log duration millisecond.

dan kalo time.Since doang setau saya malah nanosecond, bukan millisecond.
readmore: https://golang.org/pkg/time/#Duration